server/routes/auth.js

```diff
@@ -1,20 +1,20 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { validateRegister, validateLogin } = require('../validate');
-const protect = require('../middleware/auth');
+const { validateRegister, validateLogin } = require("../validate");
+const protect = require("../middleware/auth");
 const {
   registerUser,
   loginUser,
   loadUser,
   logoutUser,
-} = require('../controllers/auth');
+} = require("../controllers/auth");
 
-router.route('/register').post(validateRegister, registerUser);
+router.route("/register").post(validateRegister, registerUser);
```

Comment:
I notice we don't have prettier config file in the `/server` folder so it's linting to this by default on your local machine. We might want to make this consistent :)

Reply:
I think we can setup prettier for the server too 😄